Controller/Process/Json.php

```diff
@@ -69,6 +74,13 @@ public function __construct(
         $this->_resultFactory = $context->getResultFactory();
         $this->_adyenHelper = $adyenHelper;
         $this->_adyenLogger = $adyenLogger;
+        $this->postParams = json_decode(file_get_contents('php://input'), true);
+        
+        // Fix for Magento2.3 adding isAjax to the request params
+        if (interface_exists("\Magento\Framework\App\CsrfAwareActionInterface") && !isset($this->getRequest()->getParams()['version'])) {
```

Comment:
better to change similar to magento implementation https://github.com/magento/magento2/blob/2.3.0/app/code/Magento/Backend/App/Request/BackendValidator.php#L99
```suggestion
if(interface_exists("\Magento\Framework\App\CsrfAwareActionInterface")) {
  $request = $this->getRequest();
  if ($request instanceof HttpRequest && $request->isPost()) {
        $request->setParam('isAjax', true);
   }
 }
```
and
```
use Magento\Framework\App\Request\Http as HttpRequest;
```